industrial_ci/src/docker.env

```diff
@@ -34,6 +34,7 @@ ROS_DISTRO
 ROS_LOG_DIR
 ROS_PARALLEL_JOBS
 ROS_PARALLEL_TEST_JOBS
+_SSH_PRIVATE_KEY
```

Comment:
There is no need for this variable, you have to set-up everthing outside docker

Reply:
Is it so when I want to use this variable in industrial_ci's process (e.g. during `wstool update` in `setup_rosws`)?

Reply:
The key will be provided by the ssh agent.

Reply:
`SSH_PRIVATE_KEY` is defined on gitlab-ci host os. In order for `wstool` to access a repo via ssh on Docker, this needs to be passed.

With the updated commit, I confirmed that Docker can access the external repo that requires the ssh key.

Reply:
>  In order for wstool to access a repo via ssh on Docker, this needs to be passed.


No, it gets passed via ssh-agent. (which gets setup in #92).
Your `inject_sshkey` gets called before entering the ROS docker container.

Reply:
>  Your `inject_sshkey` gets called before entering the ROS docker container.

Even though [it's moved in to source_tests.sh](https://github.com/ros-industrial/industrial_ci/pull/220/files#diff-5e81b8b00dc0b1bfa5dcb392b8b6ec45R91)?

Reply:
Now the ssh-agent gets spawned inside docker.
But we all already have two ways to pass SSH keys.
There's no reason to add a 3rd one.
It is very easy to spawn the agent outside.

Reply:
Yeah I totally forgot #92. Then I'm not sure how to use that. Will figure it out and open a PR for documentation hopefully.